internal/plugintest/helper.go

```diff
@@ -124,6 +126,50 @@ func (h *Helper) NewWorkingDir(ctx context.Context) (*WorkingDir, error) {
 		return nil, fmt.Errorf("unable to create terraform-exec instance: %w", err)
 	}
 
+	err = tf.SetDisablePluginTLS(true)
```

Comment:
I looked at https://pkg.go.dev/github.com/hashicorp/terraform-exec@v0.16.1/tfexec but I can't find exactly what setting that `TF_DISABLE_PLUGIN_TLS` means.

Does it mean that we want TF to not fetch providers using `https`?

Reply:
This should help clarify it slightly:

https://github.com/hashicorp/terraform/blob/01b22f4b7688c9cf084214ef1dfe51ed7719fe42/internal/command/meta_providers.go#L26-L31